ui/src/components/StatisticTable.vue

```diff
@@ -28,7 +29,7 @@
             </b-td>
             <b-td class="header">{{name1}}</b-td>
             <b-td v-for="(name2, index2) in names" :key="name2" :class="{'main-diagonal': index1 === index2}">
-              <span v-if="table[name2][name1].all > 0" class="clickable-text">
+              <span v-if="table[name2][name1].all > 0" class="clickable-text" :data-e2e-testid="`table${index2}${index1}`">
```

Comment:
I suggest to add the name here to the data attribute instead of an arbitrary number (index). That means as well that the Cypress test needs to enumerate based on keys instead of iteration by number. This approach is better as the depth of the matrix is not known anyway (depends on the workspace loaded hence dynamic data). See corresponding comment in other repo/pull request.